examples/advanced/example.js

```diff
@@ -734,9 +737,12 @@ export default class Example extends Component<
         if (
           !['movePosition', 'extruding', 'rotating', 'translating', 'scaling'].includes(editType)
         ) {
+          const updatedDataInfo = updatedData.features.map(
+            feature => `${feature.geometry.type}(${getPositionCount(feature.geometry)})`
+          );
           // Don't log edits that happen as the pointer moves since they're really chatty
           // eslint-disable-next-line
-          console.log('onEdit', editType, editContext);
+          console.log('onEdit', editType, editContext, JSON.stringify(updatedDataInfo));
```

Comment:
Isn't this going to have a perf issue?

Reply:
Maybe add an on/off option to ui?

Reply:
We ignore the really chatty edits (e.g. ones that happen as the mouse is moving), so this should only happen when they click or when the start or stop dragging.

Reply:
We ignore the really chatty edits (e.g. ones that happen as the mouse is moving), so this should only happen when they click or when the start or stop dragging.